### PR TITLE
Style project

### PR DESCRIPTION
**Pull request by charlesgobina**
For this milestone, I
* Styled the project

### CODE REVIEW

File: app/javascript/components/Greeting.js [hunk 1]
```diff
@@ -14,8 +15,10 @@ const Greeting = () => {
 
   return (
     <React.Fragment>
-      <p>{ message }</p>
-      <button onClick={() => dispatch(getGreeting())}>Fetch Greetings</button>
+      <div className="greeting">
+        <h1>{ message }</h1>
+        <button onClick={() => dispatch(getGreeting())}>Greet</button>
```
Comment (Lordkaito) [inline app/javascript/components/Greeting.js:20]: - [ ] Right now, if I click the button to display the greeting, it shows me an error in the console about a type error, you may have missed something when setting your data. Here is a screenshot of the error:
Captura de pantalla_2022-05-08_00-52-08
So please work on this issue.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @charlesgobina, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

- [ ] Please include [this](https://github.com/microverseinc/readme-template/blob/master/README.md) template on your readme file and fill it with your own instructions on how to use the project, as this is part of the general requirements for the project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/javascript/components/Greeting.js (hunk 1 above)
Comment (charlesgobina) [reply app/javascript/components/Greeting.js:20]: Hello @Lordkaito,

Thank you for the feedback. I don't get the error you are getting while running it on my local machine. Could you please have another look at the project? Below is a screenshot showing my logs.

Screenshot from 2022-05-08 00-19-40

**Review by Meltrust — APPROVED** [review]
Hi @charlesgobina,

## Wow, you did it 🎉 
image


Brilliant

_Note: In order for the project to work, records need to be created on the console.  A form to create the greetings was not a requirement._

### To highlight:

- The app works✔️ 
- Nice controllers✔️ 
- Nice components✔️ 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**